ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.13.yaml

```diff
@@ -457,7 +457,37 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
-      timeout: 3h0m0s
+      timeout: 1h0m0s
+    - as: serverless-kitchensink-e2e-broker
+      commands: make test-kitchensink-e2e-single-testonly TEST=TestBroker
+      from: serverless-operator-src
+      resources:
+        limits:
+          memory: 6Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+      timeout: 2h0m0s
+    - as: serverless-kitchensink-e2e-channel
+      commands: make test-kitchensink-e2e-single-testonly TEST=TestChannel
+      from: serverless-operator-src
+      resources:
+        limits:
+          memory: 6Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+      timeout: 2h0m0s
+    - as: serverless-kitchensink-e2e-flow
+      commands: make test-kitchensink-e2e-single-testonly TEST=TestFlow
```

Comment:
We should also add `commands: make test-kitchensink-e2e-single-testonly TEST=TestSource` ?